src/IssueViz.Security.UnitTests/OpenInIDE/Http/OwinPipelineProcessorTests.cs

```diff
@@ -49,36 +48,28 @@ public void MefCtor_CheckIsExported()
         [TestMethod]
         public void MefCtor_CheckIsExported_RequestHandlersAreImported()
         {
-            var requestHandler1 = CreateHandler("/path1");
-            var requestHandler2 = CreateHandler("/path2");
-
-            var batch = new CompositionBatch();
-
-            batch.AddExport(MefTestHelpers.CreateExport<IOwinPathRequestHandler>(requestHandler1));
-            batch.AddExport(MefTestHelpers.CreateExport<IOwinPathRequestHandler>(requestHandler2));
-            batch.AddExport(MefTestHelpers.CreateExport<ILogger>());
-
+            var handler1Export = CreateHandlerExport("/path1");
```

Comment:
Changed to use the newly-public `MefTestHelpers.Compose` method.
This means this test and the one below don't need to know about compositions, batches or type catalogs.